manage.py

```diff
@@ -1,8 +1,8 @@
-#!/usr/bin/env python
 import os
 import sys
 
 if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "notesserver.settings.dev")
```

Comment:
Why is there a need to modify this file?

Reply:
I was copied it from insights dashboard repo
os.environ.setdefault("DJANGO_SETTINGS_MODULE", "analytics_dashboard.settings.local")
https://github.com/openedx/edx-analytics-dashboard/blob/master/manage.py
but, checking if we can remove this by making changes and rebuilding